style.css

```diff
@@ -0,0 +1,82 @@
+body {
+  background: rgb(221, 213, 221);
+}
+
+.block {
+  max-width: 500px;
+  margin: auto;
+}
+.survey-form {
```

Comment:
kindly use HTML and CSS best practices because stylelint won't pass this line

Reply:
[OPTIONAL] Indeed 👁️ it takes a good eye to see that `sylelint` will throw an error for this line but once again add some more context as to why it won't, how to fix it, and maybe also some documentation to the linters repo. 😉